### ISSUE TITLE
will support as a syslog (tcp/udp)server?

### ISSUE THREAD

**Issue by tqangxl** [hyperdxio/hyperdx#59]
if support file   syslog  statsd  stdin，network device(Firewall 、Switch)syslog ，os syslog  can be analyzed quickly

**Comment by wrn14897** [comment]
Good point. I think currently you can add custom receivers to the otel collector file (https://github.com/hyperdxio/hyperdx/blob/1ec122c1be033f4030da7f034d083a066e3d3b3f/docker/otel-collector/config.yaml#L1). We should def document this to help people to set it up
1. Syslog: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/syslogreceiver
2. UDP: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/udplogreceiver
3. TCP: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/tcplogreceiver
